### PR TITLE
Update nightly-rpm-builds.yml

### PR DESCRIPTION
**Pull request by msaju**
Added release-9 builds also

### CODE REVIEW

**Review by deepshikhaaa — APPROVED** [review]

File: jobs/nightly-rpm-builds.yml [hunk 1]
```diff
@@ -26,6 +26,24 @@
 
             CENTOS_VERSION=7
 
+            CENTOS_ARCH=x86_64
+      - trigger-builds:
```
Comment (obnoxxx) [inline jobs/nightly-rpm-builds.yml:30]: There are yaml check errors in this PR.
I think the alignment of this block is not quite correct: This line is indented more than the other `-trigger-builds:` lines.

https://github.com/gluster/centosci/pull/136/checks?check_run_id=1872130444

https://travis-ci.com/github/gluster/centosci/builds/216666399

**Review by obnoxxx — CHANGES_REQUESTED** [review]
Error in yaml format.

While we're at it, I think the commit message can be improved.
First, the main subject line of `Update nightly-rpm-builds.yml` is rather meaningless, secondly, this patch is not "also" adding release-9 builds, but that's the only thing it is doing. 

Suggest to change the commit message to something like:

```
nightly-rpm-builds: add release-9 builds
```

File: jobs/nightly-rpm-builds.yml (hunk 1 above)
Comment (msaju) [reply jobs/nightly-rpm-builds.yml]: Will resolve the same.

**Review by obnoxxx — APPROVED** [review]
Yaml format is good now, thanks.
Not insisting on my previous suggestion for the commit message. 😃

**Pull request merged**